frame/ethereum/src/lib.rs

```diff
@@ -315,29 +315,24 @@ pub mod pallet {
 
 	/// Current building block's transactions and receipts.
 	#[pallet::storage]
-	#[pallet::getter(fn pending)]
 	pub(super) type Pending<T: Config> =
 		StorageValue<_, Vec<(Transaction, TransactionStatus, Receipt)>, ValueQuery>;
```

Comment:
Sorry to comment on an old merged PR. I was having issues with a migration to polkadot-v1.1.0 and I was looking for the breaking change. 

I think here, `super` should have been removed too, in order to open access to `Pending` storage.